pkg/arvo/app/hood.hoon

```diff
@@ -151,6 +151,7 @@
 ++  poke-drum-unlink          (wrap poke-unlink):from-drum
 ++  poke-drum-exit            (wrap poke-exit):from-drum
 ++  poke-drum-start           (wrap poke-start):from-drum
+++  poke-drum-safemode        (wrap poke-safemode):from-drum
```

Comment:
I don't like that the action that starts the default apps is `[%poke %drum-safemode]`, that seems like the kind of misnomer that could be very confusing in the future.